zstacklib/zstacklib/utils/qmp.py

```
@@ -84,3 +136,177 @@ def qmp_subcmd(qemu_version, s_cmd):
+
+        @param qmp_cmd: QMP command to be sent as a Python dict
+        @return QMP response as a Python dict or None if the connection has
+                been closed
+        """
+        self.logger.debug(">>> %s", qmp_cmd)
+        try:
+            self.__sock.sendall(json.dumps(qmp_cmd).encode('utf-8'))
+        except OSError as err:
+            if err.errno == errno.EPIPE:
+                return None
+            raise err
```

Ruff (0.14.8)

Use `raise` without specifying exception name

Remove exception name

(TRY201)